about.html

```diff
@@ -0,0 +1,170 @@
+              <li><a class="link" href="#">News</a></li>
+              <li><a class="static-link" href="#">Health campaign</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+
+      <section class="home-section">
+        <div class="home-section_presentation">
+          <div class="intro">
+            <h2 class="h2">
+              Fit-Fam National Summits <br />
+              FFNS 2022
+            </h2>
+          </div>
+
+          <div class="about_top">
+          <p class="about-paragraph">
+            Participant will get an opportunity to partcipate in various
+            pre-conferences, panel discussions, lectures, and hands-on
+            workshops, coverind a wide range of topics and disciplines.
+          </p>
+          <div class="about_contact">
+            <small>
+              please contact us via Email for any further questions about FF
+              National Summits!
+            </small>
+            <p class="text_mail">ffsummits2022@ffnaija.org</p>
+            <hr class="mail-line" />
+          </div>
+        </div>
+      </div>
+
+        <!-- Programs -->
+        <article class="about-section">
+          <div class="logo-text-div">
+            <h2 class="logo-text">Fit-Fam National Summits 2022 Logo</h2>
+            <hr class="about_hr" />
+            <small class="small_paragraph"
+              >Lorem ipsum dolor sit amet consectetur adipisicing elit. At
+              quisquam cupiditate nam animi veritatis eum. Molestias doloremque
+              asperiores dolorum iusto Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel, excepturi!</small
+            >
+          </div>
+
+          <div class="about_logo"><img src="./img/fitfam.jpg" alt="" /></div>
+          <hr />
+
+          <div class="logo-text-div header">
+            <h2 class="logo-text">See the past F-F National summits</h2>
+            <hr class="about_hr" />
+            <small class="small_paragraph"
+              >Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quis
+              quas rem placeat deleniti unde aut sapiente nostrum, laudantium et amet.
+              delectus.</small
+            >
+          </div>
+        </article>
+
+        <article class="picture_presentation">
+          <div class="previous_events">
+            <div class="event_image">
+              <img
+                src="./img/fit-women-conference-2021.jpg"
+                alt="past-event-img"
+              />
+              <span class="conference_year year">2015</span>
+              <span class="conference_venue venue"
+                >Dominion City Ajah Lagos Nigeria</span
+              >
+            </div>
+            <div class="event_image2">
+              <img
+                src="./img/fit-women-conference-2021.jpg"
+                alt="past-event-img"
+              />
+            </div>
+          </div>
+        </article>
+      </section>
```

Comment:
Kindly, let's make these presentation pictures as we have it in the provided design:

- [ ] with a with text in the middle of the image;
- [ ] with a red filter on it, to match as much as possible the design;
- [ ] with 2 different images;

Let's see this as a very good opportunity to learn CSS's secrets :+1: 

Your design:
![image](https://user-images.githubusercontent.com/83596135/163588354-172a028a-a3b4-48e0-8ca2-7df0ac754123.png)

Provided design:
![image](https://user-images.githubusercontent.com/83596135/163588420-59634c3d-59c4-46f4-b878-18ded50a0434.png)